bot-lib/src/commands/delete_class_category.rs

```diff
@@ -0,0 +1,49 @@
+use crate::commands::find_channels;
+use crate::data::PoiseContext;
+use color_eyre::eyre::{OptionExt, Result};
+use regex::Regex;
+
+#[poise::command(
+    slash_command,
+    required_permissions = "MANAGE_CHANNELS",
+    description_localized("en-US", "Deletes a class category")
```

Comment:
Not that it really matters much, but we can just use `description` rather than `description_localized`.

Reply:
[I see it here](https://docs.rs/poise/latest/poise/macros/attr.command.html), but if other code uses it, it's fine for now.

Reply:
@ChaseParate The description flag is only available for the parameters, not the top level